Makefile

```diff
@@ -33,8 +33,8 @@ BUILDTYPE_LOWER := $(shell echo $(BUILDTYPE) | tr '[A-Z]' '[a-z]')
 EXEEXT := $(shell $(PYTHON) -c \
 		"import sys; print('.exe' if sys.platform == 'win32' else '')")
 
-NODE ?= ./node$(EXEEXT)
 NODE_EXE = node$(EXEEXT)
+NODE ?= ./$(NODE_EXE)
```

Comment:
It's not strictly necessary. Yes, they're meant to create the same result. The purpose of that change was to reduce duplication since the default value for `NODE` literally duplicates the entire right hand side of `NODE_EXE` and prefixes it. I just consolidated it to incorporate the value via variable expansion instead of literal duplication.